### DISCUSSION TITLE
Hello, how do I turn this feature off?

### DISCUSSION THREAD

**Discussion by tungdt996** [romkatv/powerlevel10k#2736] [General]
<img width="1676" alt="image" src="https://github.com/user-attachments/assets/1a2fb431-edcc-41ab-b2b0-3fba991ac7a3">

**Comment by romkatv** [comment]
What happens if you disable powerlevel10k?

**Reply by tungdt996** [reply]
Are not. I just don't want to use the highlighting function when clicking on a command frame. powerlevel10k is very good

**Reply by tungdt996** [reply]
I don't want to use the function of displaying a green frame when clicking the mouse in the image

**Comment by romkatv** [comment]
Does this function disappear if you disable powerlevel10k?

**Comment by tungdt996** [comment]
oh. I tried uninstall all end reinstall, it doesn't happen anymore. Many thanks for your support.

**Comment by romkatv** [comment]
The feature you disliked is provided by your terminal, not by powerlevel10k. I don't know why it got turned on and then turned off again, but it's not powerlevel10k doing that.
